payloads/external/edk2/Kconfig

```diff
@@ -299,4 +299,17 @@ config EDK2_CUSTOM_BUILD_PARAMS
 
 	  This option can support both macros `-D` and Pcds `--pcd`.
 
+config EDK2_CPU_THROTTLING_THRESHOLD_OPTION
+	bool "Display CPU throttling threshold option"
+	default n
+	help
+	  Enables setting a custom CPU throttling threshold temperature
+
+config CPU_MAX_TEMPERATURE
```

Comment:
These can be moved to soc/intel/{tiger,alder,meteor}lake, TjMax is generally the same for each SKU of a given SoC.

Reply:
Moved [here](https://github.com/Dasharo/coreboot/pull/513/files#diff-27ab80802178f58b3844edbdca81af8d572a5f9f1694d846fff49e0ddbdb9888R564)